client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StepContainer from '../step-container';
+import { hasNonEmptyLeaves } from 'woocommerce/woocommerce-services/lib/utils/tree';
+import { toggleStep, confirmCustoms } from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+	isCustomsFormStepSubmitted,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+import StepConfirmationButton from '../step-confirmation-button';
+import Checkbox from 'woocommerce/woocommerce-services/components/checkbox';
+
+const CustomsStep = ( props ) => {
+	const {
+		errors,
+		expanded,
+		translate,
+		isSubmitted,
+		dummyFieldChecked,
+	} = props;
+	const summary = hasNonEmptyLeaves( errors ) ? translate( 'Customs information incomplete' ) : translate( 'Customs information valid' );
+
+	return (
+		<StepContainer
+			title={ translate( 'Customs' ) }
+			summary={ isSubmitted ? summary : '' }
+			expanded={ expanded }
+			toggleStep={ props.toggleStep }
+			isSuccess={ isSubmitted && ! hasNonEmptyLeaves( errors ) }
+			isError={ isSubmitted && hasNonEmptyLeaves( errors ) } >
+			<Checkbox checked={ Boolean( dummyFieldChecked ) } onChange={ props.toggleDummyField } /> Check this to make validation pass.
+			<StepConfirmationButton
+				disabled={ hasNonEmptyLeaves( errors ) }
+				onClick={ props.confirmCustoms } >
+				{ translate( 'Confirm' ) }
```

Comment:
I'm not sure that 'Confirm' is really meaningful to the user as the button text. How about something like 'Save customs form'?

Reply:
Absolutely! Changed it in https://github.com/Automattic/wp-calypso/pull/24975/commits/ab9a74d171039b2a6b0fbbc33b20e17702ce92cc